installation/single_machine_docker.rst

```diff
@@ -16,18 +16,18 @@ This section describes setting up a Citus cluster on a single machine using dock
 
 .. code-block:: bash
 
-  curl -sSL https://get.docker.com/ | sudo bash
-  sudo usermod -aG docker $USER && newgrp docker
+  curl -sSL https://get.docker.com/ | sh
```

Comment:
FWIW I get this right up front when I run this command:
``
modprobe: FATAL: Module aufs not found in directory /lib/modules/4.4.0-64-generic
``

Installation appears to work fine, but its a bit disconcerting to see that message.